install.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+cd /vagrant
+
+gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
+curl -L https://get.rvm.io | bash -s stable
+
+source $HOME/.rvm/scripts/rvm
```

Comment:
the source line is automatically added via nvm, but the `use` statement certainly can be added. will do!